draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1072,199 @@ The following describes each member of the `concise-bom-tag` map.
   The `$$concise-bom-tag-extension` extension socket is empty in this
   specification.
 
+# CoRIM-based Evidence Verification
+
+The verification procedure is divided into three separate phases:
+
+* Appraisal Context initialisation
+* Evidence collection
+* Evidence appraisal
+
+At a few well-defined points in the procedure, the Verifier behaviour will depend on the specific CoRIM profile.
+Each CoRIM profile MUST provide a description of the expected Verifier behavior for each of those well-defined points.
+
+In particular, it is expected that the resources used during the initialisation phase can be amortised across multiple appraisals.
+Verifiers claiming compliance with this specification must exhibit the same externally visible behavior as described here, they are not required to use the same internal data structures.
+
+In the following, if a MUST-level requirement is violated, the entire procedure is aborted.
+
+## Appraisal Context initialisation
+
+The goal of the initialisation phase is to load the CoRIM Appraisal Context with objects such as CoRIM tags, cryptographic validation key material (e.g., raw public keys, root certificates, intermediate CA certificate chains), etc. that will be used in the subsequent Evidence Appraisal phase.
+
+### CoRIM Selection
+
+All available CoRIMs are collected. A Verifier may be pre-configured with a large number of tags describing many types of device. All CoRIMs are loaded at this stage, later stages will select the CoRIMs appropriate to the Evidence Appraisal step.
+
+CoRIMs that are not within their validity period, or that cannot be associated with an authenticated and authorised source MUST be discarded.
+
+CoRIM that are secured by a cryptographic mechanism such as a signature MUST be discarded if the signature chain does not validate successfully.
+
+Other selection criteria MAY be applied.
+
+For example, if the Evidence format is known in advance, tags that do not match the expected profile can be readily discarded.
+
+The selection process MUST yield at least one usable tag.
+
+### CoBOM Extraction
+
+CoBOMs are processed after all tags have been loaded from CoRIMs. All the available Concise Bill Of Material tags (CoBOMs) are collected from the selected CoRIMs.
+
+The verifier MUST check each CoBOM to see whether it can be activated. The verifier MUST activate all tags referenced by an activated CoBOM.
+
+After the verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
+
+### Tags Identification and Validation
+
+The verifier chooses tags -- including Concise Module ID Tags ([CoMID]({sec-comid})), Concise Software ID Tags ([CoSWID]), and/or Concise Trust Anchor Stores ([CoTS](I-D.ietf-rats-concise-ta-stores)) -- from the selected CoRIMs.
```

Comment:
I see you are still a young Padawan in the art of [kramdown-rfc references](https://github.com/cabo/kramdown-rfc#references) :-)

```suggestion
The verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}), Concise Software ID Tags (CoSWID, {{-coswid}}), and/or Concise Trust Anchor Stores (CoTS, {{I-D.ietf-rats-concise-ta-stores}}) -- from the selected CoRIMs.
```